server/gameconnection.py

```diff
@@ -478,10 +494,14 @@ def player(self, val: Player):
             if len(self.game.mods.keys()) > 0:
                 async with self._db.acquire() as conn:
                     uids = list(self.game.mods.keys())
-                    await conn.execute(text(
-                        """ UPDATE mod_stats s JOIN mod_version v ON v.mod_id = s.mod_id
-                            SET s.times_played = s.times_played + 1 WHERE v.uid in :ids"""),
-                        ids=tuple(uids)
+                    await conn.execute(
```

Comment:
Personally I wouldn't touch the modvault stuff. It's legacy code that's only used by the Python client and doesn't have good tests. 

If you were trying to replicate the query exactly I think it would be something like:
```python3
mod_stats.join(mod_version).update().values(
    times_played=mod_stats.c.times_played + 1
).where(mod_version.c.uid.in_uids))
```

SQLAlchemy will take care of the join condition as long as the table objects have been set up correctly.

Reply:
Actually, nvm, I just read https://docs.sqlalchemy.org/en/13/core/tutorial.html#multiple-table-updates. I feel like that way would be better, but I have no idea how `update from` compares performance wise

Reply:
I'll revert everything that touches the mod tables I think^^ I've come to rely on the tests a lot.